### PR TITLE
better bpf logging

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

**What this PR does / why we need it**:

This is the first step to provide more readable and meaningful log. 

The following work should to design some Format String to represent svc id, etc. And to do efficient replace in userspace.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: bpf/kmesh/workload/include/endpoint.h [hunk 1]
```diff
@@ -29,7 +29,13 @@ static inline int endpoint_manager(
     ret = backend_manager(kmesh_ctx, backend_v, service_id, service_v);
     if (ret != 0) {
         if (ret != -ENOENT)
-            BPF_LOG(ERR, ENDPOINT, "backend_manager failed, ret:%d\n", ret);
+            BPF_LOG(
+                ERR,
+                ENDPOINT,
+                "backend_manager failed, svc %u backend %u ret:%d\n",
```
Comment (LiZhenCheng9527) [inline bpf/kmesh/workload/include/endpoint.h:35]: What does the value of `ret` mean? 
Is it possible to delete it?

**Comment by LiZhenCheng9527** [comment]
nit:
why 
```
 BPF_LOG(
            ERR, SERVICE, "locality loadbalance match nothing in STRICT mode, service_id %d ret:%d\n", service_id, ret);
```
not
```
 BPF_LOG(ERR, SERVICE, "locality loadbalance match nothing in STRICT mode, service_id %d ret:%d\n", service_id, ret);
```

**Comment by hzxuzhonghu** [comment]
It is formatted by clang

File: bpf/kmesh/workload/include/endpoint.h (hunk 1 above)
Comment (hzxuzhonghu) [reply bpf/kmesh/workload/include/endpoint.h:35]: we need to print the return value to help debug

File: bpf/kmesh/workload/include/service.h [hunk 2]
```diff
@@ -87,13 +92,14 @@ lb_locality_failover_handle(struct kmesh_context *kmesh_ctx, __u32 service_id, s
...
+            DEBUG, SERVICE, "locality lb failover select endpoint [%u/%u/%u]", service_id, i, endpoint_k.backend_index);
         ret = endpoint_manager(kmesh_ctx, endpoint_v, service_id, service_v);
-        if (ret != -ENOENT)
-            break;
+        break;
     }
 
     if (ret)
-        BPF_LOG(ERR, SERVICE, "lb_locality_failover_hdl [%u %u %u]\n", service_id, i, endpoint_k.backend_index);
+        BPF_LOG(ERR, SERVICE, "locality lb failover [%u:%u] failed\n", service_id, kmesh_ctx->ctx->user_port);
```
Comment (nlgwcy) [inline bpf/kmesh/workload/include/service.h:102]:
```suggestion
        BPF_LOG(ERR, SERVICE, "locality lb failover [%u:%u] failed:%d\n", service_id, kmesh_ctx->ctx->user_port, ret);
```

Comment (hzxuzhonghu) [reply bpf/kmesh/workload/include/service.h:102]: ok

**Pull request merged**